src/middleware/packages/auth/services/auth.local.js

```diff
@@ -40,7 +40,7 @@ const AuthLocalService = {
   },
   actions: {
     async signup(ctx) {
-      const { username, email, password, ...rest } = ctx.params;
+      const { username, email, password, interactionId, ...rest } = ctx.params;
```

Comment:
What are the interactionIds used for or where are they specified?

Reply:
That's something completely custom because the OIDC provider doesn't take into account (at least to my knowledge) the possibility of having the login/signup form in a different frontend, communicating through API. So the mechanism of session cookies (where usually we will find the interactionId) doesn't work.